Documentation/install/upgrade.rst

```diff
@@ -7,12 +7,165 @@ Upgrade Guide
 Kubernetes Cilium Upgrade
 =========================
 
-Cilium should be upgraded using Kubernetes rolling upgrade functionality in order to minimize network disruptions for running workloads.
+Cilium should be upgraded using Kubernetes rolling upgrade functionality in
+order to minimize network disruptions for running workloads.
+
+If you have followed the installation guide from :ref:`ds_deploy`, you probably
+have deployed a single ``cilium.yaml`` file. That file contains all the
+necessary components to run Cilium in your Kubernetes cluster. Those components
+were a `ConfigMap`, a ``ServiceAccount``, a ``DaemonSet``, and the ``RBAC`` for
+Cilium to access the Kubernetes api-server.
+
+Since Cilium might need more, or less permissions to access Kubernetes
+api-server between releases, the ``RBAC`` might change between versions as well.
 
 The safest way to upgrade Cilium to version "\ |SCM_BRANCH|" is by updating the
-RBAC rules and the DaemonSet file provided, which makes sure the ConfigMap,
-initially set up by ``cilium.yaml``, already stored in the cluster will not be
-affected by the upgrade.
+``RBAC`` rules and the ``DaemonSet`` file in separate, which makes sure the
```

Comment:
"file in separate" --> "files separately"

Reply:
Done